docs/shared/upp.yaml

```diff
@@ -0,0 +1,41 @@
+upp:
+  execution:
+    batchargs:
+      export: NONE
+      nodes: 1
+      walltime: "00:05:00"
+    envcmds:
+      - module use /path/to/modulefiles
+      - module load runtime-module
+      - source /etc/profile.d/slurm.sh
+    executable: /path/to/upp.x
+    mpiargs:
+      - "--ntasks $SLURM_CPUS_ON_NODE"
+    mpicmd: srun
+  files_to_copy:
+    postxconfig-NT.txt: /path/to/postxconfig-NT.txt
+  files_to_link:
+    eta_micro_lookup.dat: /path/to/nam_micro_lookup.dat
+    params_grib2_tbl_new: /path/to/params_grib2_tbl_new
+  namelist:
+    update_values:
```

Comment:
Does it make sense to add `base_file` here to show a complete example to the user?

Reply:
Good idea, thanks, will do.